src/lib.rs

```diff
@@ -480,7 +480,7 @@ impl Api {
         let mut req = try!(Multipart::from_request(r));
 
         for &(k, ref v) in p.get_params().into_iter() {
-            req.write_text(k, v);
+            try!(req.write_text(k, v));
```

Comment:
Can you add `try!()` around the two `req.write_*` in the match block below to remove the warning?